breakerbp/failure_ratio.go

```diff
@@ -85,12 +95,20 @@ func NewFailureRatioBreaker(config Config) FailureRatioBreaker {
 		name:              config.Name,
 		minRequestsToTrip: config.MinRequestsToTrip,
 		failureThreshold:  config.FailureThreshold,
-		logger:            config.Logger,
+	}
+	if config.LogContext != nil {
+		failureBreaker.logContext = config.LogContext
+	} else {
+		failureBreaker.logContext = context.Background()
+	}
+	jitterRatio := 0.5
+	if config.TimeoutJitterRatio != nil {
+		jitterRatio = *config.TimeoutJitterRatio
```

Comment:
Should we maybe log a message if we get a jitter ratio that would be normalized (so if it's <= 0 or > 1)?